ruby_review_week_four.rb

```diff
@@ -1,22 +1,69 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+#variables are objects that can be stored and referred to again and again. 
 
 # 2. What Ruby method can you use to make a loop from an array?
 
-# 3. In Ruby, what are booleans?
+#.each , .map
+
+# 3. In Ruby, what are booleans? 
+#true or false 
 
 # 4. How do classes relate to objects?
 
+#classes are the blueprint for objects
+
 # 5. What is an instance variable?
 
+#they are class attributes
```

Comment:
not class attributes but object attributes, class attributes would be class variables